auth.php

```diff
@@ -60,6 +60,11 @@ class auth_plugin_saml2 extends auth_plugin_base {
         'logtofile'          => 0,
         'logdir'             => '/tmp/',
         'nameidasattrib'     => 0,
+        'flagresponsetype'   => saml2_settings::OPTION_FLAGGED_LOGIN_NONE,
+        'flagattribute'      => 'encumbered',
```

Comment:
I think  don't need this anymore as well as flagvalue. Also related strings need to be deleted as well.